source/php-frameworks/symfony.txt

```diff
@@ -0,0 +1,372 @@
+and query MongoDB by using a builder API. This mapping enables you to
+use other MongoDB features such as flexible schema design and advanced
+searches. To learn more about this library, see the
+:ref:`php-symfony-resources` section.
+
+.. _php-symfony-qs:
+
+Quick Start
+-----------
+
+This tutorial shows you how to build a web application by using Symfony, a
+PHP framework. It includes instructions on connecting to a MongoDB cluster hosted
+on MongoDB Atlas and accessing and displaying data from your database.
+
+By using MongoDB as a data store in a Symfony web application, you can
+leverage the document data model to build rich query expressions,
+allowing you to easily interact with data.
+
+.. tip::
+
+   If you prefer to connect to MongoDB by using the MongoDB PHP Library without
+   Symfony, see `Connecting to MongoDB
+   <https://www.mongodb.com/docs/php-library/current/tutorial/connecting/>`__
+   in the MongoDB PHP Library documentation.
+
+MongoDB Atlas is a fully managed cloud database service that hosts your
+MongoDB deployments. You can create your own free (no credit card
+required) MongoDB Atlas deployment by following the steps in this guide.
+
+This guide uses the MongoDB Doctrine ODM to allow you to map PHP objects to
+MongoDB documents and query MongoDB by using a builder API. 
+
+Follow the steps in this guide to create a sample Symfony web application
+that connects to a MongoDB deployment and performs a query on the
+database.
+
+Prerequisites
+~~~~~~~~~~~~~
+
+To create the Quick Start application, you need the following software
+installed in your development environment:
+
+- `PHP <https://www.php.net/downloads>`__
+- `Composer <https://getcomposer.org/download/>`__
+- `Symfony CLI <https://symfony.com/download>`__
+- A terminal app and shell. For MacOS users, use Terminal or a similar app.
+  For Windows users, use PowerShell.
+
+.. _php-symfony-atlas-cluster:
+
+Create a MongoDB Atlas Cluster
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+
+You must create a MongoDB cluster where you can store and manage your
+data. Complete the :atlas:`Get Started with Atlas </getting-started>`
+guide to set up a new Atlas account and create a free tier MongoDB
+cluster. This tutorial also demonstrates how to load sample datasets
+into your cluster, including the data that is used in the Quick Start tutorial.
+
+You provide instructions to the driver on where and how to connect to your
+MongoDB cluster by providing it with a connection string. To retrieve
+your connection string, follow the instructions in the :atlas:`Connect
+to Your Cluster </tutorial/connect-to-your-cluster/>` tutorial in the
+Atlas documentation.
+
+Save your connection string in a secure location.
+
+Install MongoDB Extension
+~~~~~~~~~~~~~~~~~~~~~~~~~
+
+To learn more about installing the MongoDB extension, see `Installing the Extension
+<https://www.mongodb.com/docs/php-library/current/tutorial/install-php-library/>`__
```

Comment:
This probably makes no difference, but the section link is https://www.mongodb.com/docs/php-library/current/tutorial/install-php-library/#installing-the-extension